--- origsrc/gtk+-2.12.11/gdk-pixbuf/io-jpeg.c	2008-07-01 08:42:44.000000000 -0500
+++ src/gtk+-2.12.11/gdk-pixbuf/io-jpeg.c	2008-07-04 18:51:03.593750000 -0500
@@ -964,7 +964,11 @@
 			int rc;			
 			
 			/* start decompression */
+#ifdef G_WITH_CYGWIN
+			cinfo->buffered_image = TRUE;
+#else
 			cinfo->buffered_image = cinfo->progressive_mode;
+#endif
 			rc = jpeg_start_decompress (cinfo);
 			cinfo->do_fancy_upsampling = FALSE;
 			cinfo->do_block_smoothing = FALSE;
