--- origsrc/gtk+-2.12.11/Makefile.am	2008-07-01 08:42:55.000000000 -0500
+++ src/gtk+-2.12.11/Makefile.am	2008-07-04 18:51:03.578125000 -0500
@@ -1,6 +1,6 @@
 ## Makefile.am for GTK+
 
-SRC_SUBDIRS = gdk-pixbuf gdk gtk modules demos tests perf contrib
+SRC_SUBDIRS = gdk-pixbuf gdk gtk modules demos perf contrib # tests
 SUBDIRS = po po-properties $(SRC_SUBDIRS) docs m4macros
 
 # require automake 1.4
