--- origsrc/gtk+-2.20.1/configure.in	2010-05-02 18:33:18.000000000 -0500
+++ src/gtk+-2.20.1/configure.in	2010-09-08 13:28:38.850706000 -0500
@@ -913,9 +913,14 @@ fi
 
 dnl Test for libpng
   if test x$with_libpng != xno && test -z "$LIBPNG"; then
-    AC_MSG_CHECKING(for libpng12)
-    if $PKG_CONFIG --exists libpng12 ; then
-        AC_MSG_RESULT(yes)
+    AC_MSG_CHECKING(for libpng version)
+    if $PKG_CONFIG --exists libpng14 ; then
+        AC_MSG_RESULT(libpng14)
+        PNG='png'
+	PNG_DEP_CFLAGS_PACKAGES=libpng14
+	LIBPNG=`$PKG_CONFIG --libs libpng14`
+    elif $PKG_CONFIG --exists libpng12 ; then
+        AC_MSG_RESULT(libpng12)
         PNG='png'
 	PNG_DEP_CFLAGS_PACKAGES=libpng12
 	LIBPNG=`$PKG_CONFIG --libs libpng12`
