introspection_files will get $(srcdir) added later

--- origsrc/gtk+-2.20.1/gdk/Makefile.am	2010-05-02 18:33:18.000000000 -0500
+++ src/gtk+-2.20.1/gdk/Makefile.am	2010-06-15 02:12:33.950175600 -0500
@@ -199,7 +199,7 @@ introspection_files = \
 		$(gdk_c_sources) \
         gdkenumtypes.c \
         gdkenumtypes.h \
-        $(srcdir)/x11/*.c
+        x11/*.c
 
 Gdk-2.0.gir: $(gdktargetlib) Makefile
 Gdk_2_0_gir_SCANNERFLAGS = --strip-prefix=Gdk --add-include-path=../gdk-pixbuf
