--- origsrc/gtk+-2.22.0/gdk/Makefile.am	2010-09-22 22:00:45.000000000 -0500
+++ src/gtk+-2.22.0/gdk/Makefile.am	2010-11-04 20:28:12.112617900 -0500
@@ -242,7 +242,7 @@ GdkX11_2_0_gir_SCANNERFLAGS = --warn-all
 GdkX11_2_0_gir_INCLUDES = Gio-2.0 Gdk-2.0 GdkPixbuf-2.0 Pango-1.0 xlib-2.0
 GdkX11_2_0_gir_LIBS = $(gdktargetlib)
 GdkX11_2_0_gir_FILES = $(x11_introspection_files)
-GdkX11_2_0_gir_CFLAGS = $(INCLUDES) -L$(top_builddir)/gdk
+GdkX11_2_0_gir_CFLAGS = $(INCLUDES) -I$(top_builddir) -I$(top_srcdir)/gdk
 INTROSPECTION_GIRS += GdkX11-2.0.gir
 
 introspection_files += $(filter-out x11/gdkx.h, $(x11_introspection_files))
