--- origsrc/gtk+-2.24.28/gtk/gtkcolorbutton.c	2014-09-29 15:02:17.000000000 -0500
+++ src/gtk+-2.24.28/gtk/gtkcolorbutton.c	2015-08-03 21:43:33.607984100 -0500
@@ -684,7 +684,7 @@ gtk_color_button_set_alpha (GtkColorButt
 /**
  * gtk_color_button_get_color:
  * @color_button: a #GtkColorButton.
- * @color: a #GdkColor to fill in with the current color.
+ * @color: (out): a #GdkColor to fill in with the current color.
  *
  * Sets @color to be the current color in the #GtkColorButton widget.
  *
